src/algebra/ordered_monoid.lean

```diff
@@ -1076,16 +1132,16 @@ fn_min_mul_fn_max id n m
 /-- A linearly ordered cancellative additive commutative monoid
 is an additive commutative monoid with a decidable linear order
 in which addition is cancellative and monotone. -/
-@[protect_proj, ancestor ordered_cancel_add_comm_monoid linear_order]
+@[protect_proj, ancestor ordered_cancel_add_comm_monoid linear_ordered_add_comm_monoid]
 class linear_ordered_cancel_add_comm_monoid (α : Type u)
-  extends ordered_cancel_add_comm_monoid α, linear_order α
+  extends ordered_cancel_add_comm_monoid α, linear_ordered_add_comm_monoid α
 
 /-- A linearly ordered cancellative commutative monoid
 is a commutative monoid with a linear order
 in which multiplication is cancellative and monotone. -/
-@[protect_proj, ancestor ordered_cancel_comm_monoid linear_order, to_additive]
+@[protect_proj, ancestor ordered_cancel_comm_monoid linear_ordered_comm_monoid, to_additive]
 class linear_ordered_cancel_comm_monoid (α : Type u)
-  extends ordered_cancel_comm_monoid α, linear_order α
+  extends ordered_cancel_comm_monoid α, linear_ordered_comm_monoid α
```

Comment:
can you change line 1185 to use `hf.linear_ordered_comm_monoid` instead of `linear_order.lift`. It doesn't make much difference, but it's nice if the `extends` clause matches the list of things we unpack.

Reply:
That works